content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -87,6 +87,16 @@ Follow these steps to get started with ambient:
     ztunnel          1         1         1       1            1           <none>                   82s
     {{< /text >}}
 
+1.  Install Kubernetes Gateway CRDs, which don’t come installed by default on most Kubernetes clusters:
```

Comment:
Move this up to the first step?

Reply:
The 1st step is to install the kind cluster, then install Istio, and verify the istio install. I think installing the CRDs is in the correct order.